en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md

```diff
@@ -412,20 +419,21 @@ powerful research tools that you couldn't use otherwise, and can serve
 as a basis for more advanced work. For the purposes of this tutorial,
 you need to learn only a few, very simple commands.
 
-First, open a command line window. If you are using a Mac, open Terminal
-in the 'Applications/Utilities' directory. On Windows, you'll use
-PowerShell. On Windows 7 or later, click Start, type "powershell" in
-"Search programs and files," and hit enter. For a detailed introduction
-to using the command line, see Zed A. Shaw's excellent *[Command Line
-Crash Course.](http://cli.learncodethehardway.org/book/)*
-
-Once opened, you should see a text window and a prompt that looks
-something like this: `computer-name:~username$`. The tilde indicates
-your "home" directory, and in fact you can type `$ cd ~` at any point to
-return to your home directory. Don't type the dollar sign, it just
-symbolizes the command prompt of your terminal, promting you to type something
-into your terminal (as opposed to typing it into your document); remember to hit
-enter after every command.
+First, open a command line window. If you are using macOS, open Terminal in the
+'Applications/Utilities' directory. On Windows, you'll use PowerShell or, better yet, the
+Windows Subsystem for Linux (tough the latter is more complex to learn). On Windows 7 or later,
+click Start, type "powershell" in "Search programs and files," and hit enter. For a detailed
+introduction to using the command line, see Zed A. Shaw's excellent *[Command Line Crash
```

Comment:
You'll see there's a merge conflict - unfortunately we had to remove the link to Zed Shaw's book because it was giving a 404 response (and also never seems to have been archived in the Wayback Machine, which is normally what we'd try to point to before actually removing a link.)

If you know of where this book moved to, by all means put in a link. We also encourage cross-linking between PH lessons, so an easy (albeit very beginner lesson) is at https://programminghistorian.org/en/lessons/intro-to-bash

Reply:
Yes, most definitely lets go with the PH tutorial!